samples/fconnection/ConnectionSupervisionTask.cpp

```diff
@@ -39,44 +36,21 @@ long ConnectionSupervisionTask::GetFilterCode() noexcept {
 void ConnectionSupervisionTask::operator()() {
     DARWIN_LOGGER;
     bool is_log = GetOutputType() == darwin::config::output_type::LOG;
+    unsigned int certitude;
 
     for (const std::string &connection : _connections) {
         SetStartingTime();
-        // We have a generic hash function, which takes no arguments as these can be of very different types depending
-        // on the nature of the filter
-        // So instead, we set an attribute corresponding to the current connection being processed, to compute the hash
-        // accordingly
+
         _current_connection = connection;
-        unsigned int certitude;
-        xxh::hash64_t hash;
-
-        if (_is_cache) {
-            hash = GenerateHash();
-
-            if (GetCacheResult(hash, certitude)) {
-                if (is_log && (certitude>=_threshold)){
-                    _logs += R"({"evt_id": ")" + Evt_idToString() + R"(", "time": ")" + darwin::time_utils::GetTime() + R"(", "filter": ")" + GetFilterName() +
-                            R"(", "connection": ")" + connection + R"(", "certitude": )" + std::to_string(certitude) + "}\n";
-                }
-                _certitudes.push_back(certitude);
-                continue;
-            }
-        }
```

Comment:
Cache is removed because we can have a non new connection in cache meanwhile the connection is removed from the redis due to the redis expiration, and the connection should be new.
I didn't find any good solution to manage that, feel free to draft this pr and bring a solution

Reply:
We already use Redis as cache, why would we need another one that is a duplicate ?